packages/esm-ward-app/src/root.component.tsx

```diff
@@ -0,0 +1,19 @@
+import React from 'react';
+import { BrowserRouter, Route, Routes } from 'react-router-dom';
+import Home from './home.comonent';
+
+const Root: React.FC = () => {
+  const wardViewBasename = window.getOpenmrsSpaBase() + 'ward';
+
+  return (
+    <main>
+      <BrowserRouter basename={wardViewBasename}>
+        <Routes>
+          <Route path="/" element={<Home />} />
```

Comment:
Per above, there is no current use case to have a Home component at the root of this app.  Our initial view that we are designing out will be a component that renders the patients who are currently assigned to a particular ward.  So let's create that.  Let's put that at route `/ward/${locationUuid}/patients/`, and renders a component defined as `<WardPatients location={locationUuid} />`.  Please create this WardPatients component.  You can render the location name in the top left corner as a starting point, per the mockups.